File: MODUL-2/bahasa.c/PRAK204-2310817210007-RaymondHariyono.c
#include <stdio.h>
#define phi 3.142857
int main(){
    float r,t;
    
    printf("Input \n");
        scanf("%f",&r);
        scanf("%f",&t);

    float v= phi * r*r * t;
    float l= 2 * phi * r *(r+t);
    float k= 2 * phi * r;
    
    printf("\n");
    printf("Output \n");

    printf("Alas = %.2f \n", v);
    printf("Tinggi = %.2f \n", l);
    printf("Keliling = %.2f", k);

return 0;
}

File: MODUL-2/python/PRAK204-2310817210007-RaymondHariyono.py
print("input")

r,t = map(float, input().split())
        
phi = 3.142857

volume = phi * r ** 2 * t
luas = 2* phi * r * (r+t)
keliling = 2 * phi * r
print()
print("Output")
print(f"Alas = {volume:.2f}")
print(f"Tinggi = {luas:.2f}")
print(f"Keliling = {keliling:.2f}")